### ISSUE TITLE
Wish: Output Directory Optional

### ISSUE THREAD

**Issue by cawoodm** [LonnyGomes/ReSignMe#10]
In my experience we mostly want the resigned file output to the same folder as the oroginal .ipa - it get's renamed anyway.

I suggest Output Directory is left empty by default and just defaults to the path of the original.

**Comment by ghost** [comment]
yup, that could work. I'll add this feature in the next release.
